Tars/Tars/ViewController/UniverseSearchViewController.swift

```diff
@@ -589,4 +603,14 @@ extension UniverseSearchViewController {
             }
         }
     }
+    
+    /// 탐색, 검색화면에서의 음소거
+    private func muteExploreModeSound(soundPlayer: [String: SCNAudioPlayer]) {
```

Comment:
p2;
상세정보로 넘어오면서 탐색/검색뷰의 소리를 없애는 역할인 것으로 보이는군요!
만약 탐색/검색 화면을 모두 포함하는 것이라면 이름을 살짝 변경하면 어떨까 싶습니다.
```suggestion
 private func muteExploreSearchModeSound(soundPlayer: [String: SCNAudioPlayer]) {
```
이런식으로 이름을 조금 수정하면 어떨까 의견을 내봅니다 🙂

Reply:
https://github.com/DeveloperAcademy-POSTECH/MacC-Team-TARS/pull/86/commits/93305f44d6efd7aa441e755976158d43d7b15d5d 반영했습니다. 의견 감사합니다 !!